benchmark/sparse_blas/operations.cpp

```diff
@@ -726,9 +826,25 @@ const std::map<std::string,
          [](const Mtx* mtx) {
              return std::make_unique<SymbolicCholeskyOperation>(mtx, false);
          }},
-        {"symbolic_cholesky_symmetric", [](const Mtx* mtx) {
+        {"symbolic_cholesky_symmetric",
+         [](const Mtx* mtx) {
              return std::make_unique<SymbolicCholeskyOperation>(mtx, true);
-         }}};
+         }},
+        {"reorder_rcm",
+         [](const Mtx* mtx) {
+             return std::make_unique<ReorderRcmOperation>(mtx);
+         }},
+        {"reorder_amd", [](const Mtx* mtx) {
+             return std::make_unique<ReorderApproxMinDegOperation>(mtx);
+         }},
+#if GKO_HAVE_METIS
+    {
+        "reorder_nd", [](const Mtx* mtx) {
+            return std::make_unique<ReorderNestedDissectionOperation>(mtx);
+        }
+    }
```

Comment:
same here

Reply:
Pulled the ifdef inside to make it easier for the formatter.